tests/CLI/modules/config_tests.py

```diff
@@ -62,19 +62,16 @@ def test_setup(self, mocked_input, getpass, confirm_mock):
             getpass.return_value = 'A' * 64
             mocked_input.side_effect = ['user', 'public', 0]
 
-            result = self.run_command(['--config=%s' % config_file.name,
-                                       'config', 'setup'])
+            result = self.run_command(['--config=%s' % config_file.name, 'config', 'setup'])
 
             self.assert_no_fail(result)
-            self.assertTrue('Configuration Updated Successfully'
-                            in result.output)
+            self.assertTrue('Configuration Updated Successfully' in result.output)
             contents = config_file.read().decode("utf-8")
             self.assertTrue('[softlayer]' in contents)
             self.assertTrue('username = user' in contents)
             self.assertTrue('api_key = AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'
                             'AAAAAAAAAAAAAAAAAAAAAAAAAAAAA' in contents)
```

Comment:
The test_setup unittest failed, it seems you worked on windows and that's why you did not trip over this issue
https://travis-ci.org/softlayer/softlayer-python/jobs/460519115#L598